### DISCUSSION TITLE
Cargo shipment papers

### DISCUSSION THREAD

**Discussion by SweptWasTaken** [space-wizards/space-station-14#4036] [Feature Request]
Crates order by cargo need a piece of paper attached to the crate. The paper will state what the name of the crate ordered is, who ordered it, how much it cost and what it contains.

**Comment by PaulRitter** [comment]
could also have a /vg/-ish mechanic where it would randomly send wrong amounts and you'd have to return them to get a bonus. this mechanic would also pay you a small bonus for each correctly approved order paper (stamped) which you sent back and would apply a penalty for each wrongfully approved order
